skimage/segmentation/random_walker_segmentation.py

```diff
@@ -337,6 +356,11 @@ def random_walker(data, labels, beta=130, mode='bf', tol=1.e-3, copy=True,
     This linear system is solved in the algorithm using a direct method for
     small images, and an iterative method for larger images.
 
+    References
+    ----------
+    .. [1] Leo Grady, Random walks for image segmentation, IEEE Trans Pattern
+    Anal Mach Intell. 2006 Nov;28(11):1768-83. :DOI:`10.1109/TPAMI.2006.233`.
```

Comment:
I think this line must be indented, causing the travis failure.

Reply:
thanks I was getting crazy over this failure!

Reply:
you're welcome :)

I'm not sure it is indented enough. I don't think sphinx consider indents as 4 spaces like in python, but rather an alignment with the L of Leo. At least, this is how we do elsewhere in the library.

Sphinx is particularly annoying on that, and hard to debug.

Reply:
Yes I hesitated between the two ways of indenting but Travis is all green now, so I'd prefer not over-indenting.